starter-code/css/main.css

```diff
@@ -1,8 +1,146 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+
+body {
+ text-decoration: none;
+
+}
+.navbar {
```

Comment:
Try to use this code (but of course adapt it to your class and structure) it should help you to have a nice navbar

```css

#header {
  position: fixed;
  height: 90px;
  background-color: white;
  width: 100%;
  display: flex;
  box-shadow: 0px 10px 5px -8px rgba(0, 0, 0, 0.75);
}

#header img {
  height: 70%;
  margin: 10px 0 0 30px;
}

#header ul {
  display: flex;
  align-items: baseline;
  margin: 2% 0 0 50%;
}

#header li {
  margin: 0 20px;
  list-style-type: none;
}

```